yarn-project/aztec-node/src/aztec-node/server.ts

```diff
@@ -570,6 +571,24 @@ export class AztecNodeService implements AztecNode, AztecNodeAdmin, Traceable {
     return Promise.resolve(this.l1ChainId);
   }
 
+  public getUtilityContextWithoutContractAddress(): Promise<UtilityContextWithoutContractAddress> {
+    return Promise.all([this.getBlockHeader('latest'), this.getVersion(), this.getChainId()]).then(
+      ([blockHeader, version, chainId]) => {
+        if (!blockHeader) {
+          throw new Error(
+            'Aztec Node failed to obtain the latest block header when fetching the utility context without contract address.',
+          );
+        }
+        return UtilityContextWithoutContractAddress.from({
+          blockNumber: blockHeader.globalVariables.blockNumber,
+          timestamp: blockHeader.globalVariables.timestamp,
+          version,
+          chainId,
+        });
+      },
+    );
```

Comment:
Unsure about this. Merging these in a single call is _fine_ I guess, but I'd definitely not pollute the node with the term 'utility context'.

Alternatively we can just do
```ts
const [ blockNumber, timestamp, version, chaind ] = await Promise.all([node.getBlockNumber(), node.getTimestamp(), node.getVersion(), node.getChainId()];
```

and get an equivalent result.